lib/sites-base.js

```diff
@@ -89,6 +89,43 @@ module.exports = {
         label: 'Admin Password',
         type: 'password',
         help: 'Set password for the "admin" user of the new site. For pre-existing sites, leave blank for no change.'
+      },
+      {
+        name: 'redirect',
+        label: 'Redirect Entire Site',
+        type: 'boolean',
+        help: 'Redirect all traffic for the site to another URL.',
+        showFields: {
+          true: [ 'redirectUrl', 'redirectPreservePath' ]
+        }
+      },
+      {
+        name: 'redirectUrl',
+        label: 'Redirect To...',
+        type: 'url',
+        help: 'Redirect traffic to this URL.'
+      },
+      {
+        name: 'redirectPreservePath',
+        label: 'Preserve the Path when Redirecting',
+        type: 'boolean',
+        help: 'If the URL ends with /something, add /something to the redirect URL as well. Otherwise, all traffic is redirected to a single place.'
+      },
+      {
+        name: 'redirectStatus',
+        label: 'Redirect Status Code',
+        type: 'select',
+        choices: [
```

Comment:
It looks like this needs to be required.

Reply:
It's a select element so it defaults to the first choice.

Reply:
I'd feel better we were explicit on a `def` value. Too easy for someone down the road to throw an option in the array (theoretically).